MetaMask/metamask-extension
# Expand truncated address

Fixes #20966


## **Description**
We want to change our truncated addresses to a "0x12345...12345" format to give users more detail when selecting an account to connect with dapps.

## **Manual testing steps**
1: Go to a dapp 
2. Connect to a different wallet, or click the Connected Sites icon
3. Verify that address is now formatted as "0x12345...12345"

## **Screenshots/Recordings**

_If applicable, add screenshots and/or recordings to visualize the before and after of your change._

### **Before**

![image](https://github.com/MetaMask/metamask-extension/assets/10986371/be990338-131b-4d2d-8f15-1c83fa1d8417)
### **After**

<img width="340" alt="image" src="https://github.com/MetaMask/metamask-extension/assets/10986371/3269d822-7f2a-4040-ad2f-1a64e9bfbd1b">

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
